matlab/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+find_package(Matlab REQUIRED)
+matlab_add_mex(NAME finufft_mex SRC finufft.cpp LINK_TO finufft OUTPUT_NAME finufft R2018a)
+target_compile_definitions(finufft_mex PRIVATE -DR2008OO)
+
+file(GLOB FINUFFT_MATLAB_M_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/*.m)
+file(GLOB FINUFFT_MATLAB_DOC_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/*.docsrc)
```

Comment:
the *.docsrc here are only used by my script `addmhelp.sh` which is part of the docs developer build (see `docs` target in `makefile`).    So, they are not needed.

Reply:
I guess, more generally about matlab, I find it weird to copy all *.m source files to `build/matlab`. What if those files were huge? Isn't it better to copy the MEX exec back into FINUFFT/matlab/ ?

Reply:
The build location should be conceptually separate from the source location. For example, if we were to copy it back, we would not be able to test Debug and Release builds in the same tree (you would need to make a copy of the `finufft` source). I would also expect text files to be small enough to not be an issue.